src/elements/content-sidebar/versions/__tests__/VersionsItem.test.js

```diff
@@ -238,5 +239,110 @@ describe('elements/content-sidebar/versions/VersionsItem', () => {
             expect(wrapper.exists(VersionsItemRetention)).toBe(true);
             expect(wrapper.find(VersionsItemActions).prop('isRetained')).toBe(true);
         });
+
+        test.each`
+            permissions
+            ${{ can_delete: true, can_download: true, can_preview: true, can_upload: true }}
+            ${{ can_delete: true, can_download: true, can_preview: true, can_upload: false }}
+            ${{ can_delete: true, can_download: true, can_preview: false, can_upload: true }}
+            ${{ can_delete: true, can_download: false, can_preview: true, can_upload: true }}
+            ${{ can_delete: false, can_download: true, can_preview: true, can_upload: true }}
+            ${{ can_delete: true, can_download: true, can_preview: false, can_upload: false }}
+            ${{ can_delete: true, can_download: false, can_preview: true, can_upload: false }}
+            ${{ can_delete: false, can_download: true, can_preview: false, can_upload: true }}
+            ${{ can_delete: false, can_download: false, can_preview: true, can_upload: true }}
```

Comment:
Is there any reason you test those 9 combinations instead of all 16? Same for the last test

Reply:
Here, some of those have to be tested in test below, since none of actions are available and we don't render component. Rest didn't have affected permission, but I'll add them just in case